helpers_yo

```diff
@@ -59,7 +59,12 @@ __yo_getDescription() {
 }
 
 __yo_get_installed_generators() {
-  command yo --generators | while read line; do __yo_getGeneratorMetadata "$line"; done
+  local REMOVE_TITLE="1"
+  local YO_VERSION=$(yo --version)
+  if [[ "$(printf '%s\n' "$YO_VERSION" "1.6.0" | sort -V | head -n1)" == "1.6.0" ]]; then
```

Comment:
Latest `yo` version is [`4.3.0`](https://github.com/yeoman/yo/releases/tag/v4.3.0) why the `1.6.0`?

Reply:
This line checks for all versions greater than or equal to 1.6.0 because from that version onwards, they add the available generators line at the top of the output of —generators.